js/style/style.js

```diff
@@ -15,7 +15,7 @@ var browser = require('../util/browser');
 var Dispatcher = require('../util/dispatcher');
 var Point = require('point-geometry');
 var AnimationLoop = require('./animation_loop');
-var validate = require('mapbox-gl-style-lint/lib/validate/latest');
+var validate = require('mapbox-gl-style-spec/lib/validate/latest');
```

Comment:
See the bullet point, relies on https://github.com/mapbox/mapbox-gl-style-spec/pull/259